library/scripts/adi_xilinx_device_info_enc.tcl

```diff
@@ -116,7 +116,7 @@ proc adi_device_spec {cellpath param} {
       FPGA_TECHNOLOGY {
           switch  -regexp -- $part {
              ^xc7          {set series_name 7series}
-             ^xczu         {set series_name ultrascale+}
+             ^xc[zv]u      {set series_name ultrascale+}
```

Comment:
VCU118 part is `XCVU`9P-L2FLGA2104E FPGA

Ultrascale+: XCVU3P 	XCVU5P 	XCVU7P 	XCVU9P 	XCVU11P 	XCVU13P
Ultrascale: XCVU065 	XCVU080 	XCVU095 	XCVU125 	XCVU160 	XCVU190 	XCVU440

Both share the came prefix but ultrascale+ is suffixed by P

https://www.amd.com/en/products/adaptive-socs-and-fpgas/fpga/virtex-ultrascale.html#product-table
https://www.amd.com/en/products/adaptive-socs-and-fpgas/fpga/virtex-ultrascale-plus.html#product-table

```suggestion
             ^xczu         {set series_name ultrascale+}
             ^xcvu.?.p    {set series_name ultrascale+}
```

Reply:
Great catch, thanks for the suggestion!